src/test/java/uk/gov/ida/eventemitter/TestEventEmitterModule.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.ida.eventemitter;
+
+import cloud.localstack.TestUtils;
+import com.amazonaws.services.sqs.AmazonSQS;
+import com.google.inject.AbstractModule;
+import com.google.inject.Provides;
+
+import java.util.Optional;
+
+public class TestEventEmitterModule extends AbstractModule {
```

Comment:
Why not extend the `EventEmitterModule`?
Then we've got all of the production config, overriding a couple of things if we need it.

In fact, there is also the option to use
```
Modules.override(new ProductionModule()).with(new TestModule());
```